.github/workflows/update-submodules.yml

```diff
@@ -0,0 +1,21 @@
+name: Update Theos Submodules
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.UPDATE_SUBMODULES_TOKEN }}
+
+      - name: Update submodules
+        run: |
+          git config --global user.name 'PoomSmart'
+          git config --global user.email 'poomsmart@users.noreply.github.com'
```

Comment:
Would it be best to use the @theosbot account details instead, since this presumably can be triggered by anyone in the Theos org, and the commit will be pushed under the bot’s credentials?

Reply:
Actually, I believe you should be able to use `$GITHUB_ACTOR`, like the wiki publish action does here: https://github.com/theos/github-wiki-publish-action/blob/dcd234a56f4e0bee5c284293529d21785eb9825f/entrypoint.sh#L50-L51

Reply:
Sure.